hadoop-hdds/docs/content/interface/OzoneFS.md

```diff
@@ -77,13 +77,39 @@ Or put command etc. In other words, all programs like Hive, Spark, and Distcp wi
 Please note that any keys created/deleted in the bucket using methods apart from OzoneFileSystem will show up as directories and files in the Ozone File System.
 
 Note: Bucket and volume names are not allowed to have a period in them.
-Moreover, the filesystem URI can take a fully qualified form with the OM host and port as a part of the path following the volume name.
-For example,
+Moreover, the filesystem URI can take a fully qualified form with the OM host and an optional port as a part of the path following the volume name.
+For example, you can specify both host and port:
 
 {{< highlight bash>}}
 hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:5678/key
 {{< /highlight >}}
 
+When the port number is not specified, it will be retrieved from config key `ozone.om.address`.
```

Comment:
When the port number is not specified, it will be retrieved from config key `ozone.om.address` if defined or fall back to default port 9862.

With this, we can eliminate, last lines from 109-111.

Reply:
done. still keeping L109 but repharsed.